ghost/i18n/locales/lv/comments.json

```
@@ -0,0 +1,71 @@
+{
+    "{{amount}} characters left": "Atlikušas {{amount}} rakstzīmes",
+    "{{amount}} comments": "{{amount}} komentāri",
+    "{{amount}} hrs ago": "pirms {{amount}} stundam",
+    "{{amount}} mins ago": "Pirms {{amount}} minūtēm",
+    "{{amount}} more": "Vēl {{amount}}",
+    "1 comment": "1 komentārs",
+    "Add comment": "Pievienot komentāru",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Pievienojiet savam komentāram kontekstu, kopīgojiet savu vārdu un zināšanas, lai veicinātu veselīgu diskusiju.",
+    "Add reply": "Pievienot atbildi",
+    "Already a member?": "Vai jau esat dalībnieks?",
+    "Anonymous": "Anonīms",
+    "Are you sure?": "Vai esat pārliecināts?",
+    "Become a member of {{publication}} to start commenting.": "Kļūstiet par {{publication}} dalībnieku, lai sāktu komentēt.",
+    "Become a paid member of {{publication}} to start commenting.": "Kļūstiet par apmaksātu {{publication}} dalībnieku, lai sāktu komentēt.",
+    "Cancel": "Atcelt",
+    "Comment": "komentēt",

```

```suggestion
    "Comment": "Komentēt",
```
This is a button label, so should be capitalized like the others.  (If that's not correct for Latvian, then there are a lot of other strings that are wrong.)